keylime/ca_util.py

```diff
@@ -75,6 +75,7 @@ def setpassword(pw):
 
 def cmd_mkcert(workingdir, name, password=None):
     cwd = os.getcwd()
+    mask = os.umask(0o037)
```

Comment:
Is it possible to avoid the use of umask? I see that is later reset in the `finally`, this still creates a state in the process that affect any created file, and sometimes it is difficult to track them.

IIUC you want that the files are read by the group, so why `0o640` as you did in the `fdopen` is not enough?

Reply:
The Keylime services are starting with the mask `0o077` which turns the `0o640` to `0o600` again. The alternative would be to change the mask to `0o037` in general. I don't know if that is the better option.